include/deal.II/base/mpi.h

```diff
@@ -1539,11 +1539,195 @@ namespace Utilities
     }
 
 
+    template <typename T>
+    T
+    _builtin_broadcast(const MPI_Comm    &comm,
```

Comment:
We normally rather use `namespace internal` instead of prefixing with an underscore.

Reply:
Fixed in 5a80fb6cc8996315c5328863c6f4df602a8efbe6